imap/jmap_calendar.c

```diff
@@ -7979,7 +7979,7 @@ static int jmap_calendarevent_participantreply(struct jmap_req *req)
         break;
     }
 
-    sched_request(req->accountid, NULL, organizer,
+    sched_request(req->accountid, req->accountid, NULL, organizer,
```

Comment:
Same, the schedule_id being req->accountid looks suspicious to me.

Reply:
Fixed